lib/rules/jsx-no-useless-fragment.js

```diff
@@ -79,7 +79,7 @@ function containsCallExpression(node) {
 }
 
 const messages = {
-  NeedsMoreChildren: 'Fragments should contain more than one child - otherwise, there‘s no need for a Fragment at all.',
+  NeedsMoreChildren: 'Fragments should contain more than one child - otherwise, there\'s no need for a Fragment at all.',
```

Comment:
straight quotes are always incorrect in prose; in fact, the curly single quote is the *only* correct character to use.

I do see it's pointing the wrong way tho, which means:
```suggestion
  NeedsMoreChildren: 'Fragments should contain more than one child - otherwise, there’s no need for a Fragment at all.',
```
is the proper fix.

Reply:
> straight quotes are always incorrect in prose; in fact, the curly single quote is the only correct character to use.

Today I learned! Thank you for the review 🙌